include/irr/builtin/glsl/bxdf/common.glsl

```diff
@@ -39,7 +39,12 @@ mat3 irr_glsl_getTangentFrame(in irr_glsl_AnisotropicViewSurfaceInteraction inte
 struct irr_glsl_BSDFSample
 {
    vec3 L;  // incoming direction, normalized
-   float probability; // for a single sample (don't care about number drawn)
+   float LdotT;
+   float LdotB;
+   float LdotN;
+   
+   float NdotH;
+   float VdotH;//equal to LdotH
```

Comment:
note that for diffuse BSDFs these will be exactly `NdotH==1.0` and `VdotH==VdotN`